### PR TITLE
QUEST300 37 - 60 (modified)

### PR DESCRIPTION
**Pull request by neon279**
QUEST300 20 - 29

### CODE REVIEW

**Comment by neon279** [comment]
Questions:
1. "QUEST_LV_0300_20150717_000048   인사한다" is it greeting or farewell?
2. What is English name for 메일레?
3. Is "석화서리" Petrification Frost or Stone Frost? I used Petrification Frost.
4. In QUEST_LV_0300_20150717_000056 - how many detectors at entrance? 
5. In QUEST_LV_0300_20150717_000057 and QUEST_LV_0300_20150717_000060 - how many detectors are broken? 
I just assume 4 and 5 as one detector when I was translating. Translation should be modified if there's more than one detector.

File: QUEST_LV_0300.txt [hunk 1]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000039 $Destroy Terminal(2)
+QUEST_LV_0300_20150717_000040 $Igniting explosive powder/FIRE/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000041 $Notice/Terminal is destroyed by the explosion!#5
+QUEST_LV_0300_20150717_000042 $Destroy Terminal(3)
+QUEST_LV_0300_20150717_000043 $Eternal Repose(1)
+QUEST_LV_0300_20150717_000044 $Last thing to say
+QUEST_LV_0300_20150717_000045 $Eternal Repose(2)
+QUEST_LV_0300_20150717_000046 $Placing Orb/MAKING/3/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000047 $Eternal Repose(3)
+QUEST_LV_0300_20150717_000048 $Say bye
```
Comment (stvkno) [inline QUEST_LV_0300.txt]: QUEST_LV_0300_20150717_000048 -> Say hello

File: QUEST_LV_0300.txt [hunk 2]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000049 $Notice/Svitrigaila's hope is granted#5
+QUEST_LV_0300_20150717_000050 $Check detector
+QUEST_LV_0300_20150717_000051 $Talk to Kupole 메일레
```
Comment (stvkno) [inline QUEST_LV_0300.txt]: QUEST_LV_0300_20150717_000051 -> Talk to Kupol Meile

Comment (neon279) [inline QUEST_LV_0300.txt]: Is it Kupol or Kupole? I copied it from QUEST_LV_0200.tsv

File: QUEST_LV_0300.txt (hunk 1 above)
Comment (neon279) [inline QUEST_LV_0300.txt]: I'm not sure 인사한다 is greeting or farewell.
Are you sure about this?

File: QUEST_LV_0300.txt [hunk 3]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000052 $Talk to Kupole who is rescued from Petrification Frost.
```
Comment (stvkno) [inline QUEST_LV_0300.txt]: QUEST_LV_0300_20150717_000052 -> Talk to Kupol who was saved from Stone Frost

석화서리 is translated as Stone Frost elsewhere in this repo

File: QUEST_LV_0300.txt (hunk 1 above)
Comment (Sunaries) [inline QUEST_LV_0300.txt]: "Greetings"

File: QUEST_LV_0300.txt [hunk 4]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000053 $Check Petrification detectors
+QUEST_LV_0300_20150717_000054 $To help Kupole 메일레 to guide souls safely, check detectors that you have not checked in this area.
```
Comment (stvkno) [inline QUEST_LV_0300.txt]: QUEST_LV_0300_20150717_000054 -> To help Kupol Meile guide souls safely, check the remaining detectors in the area.

File: QUEST_LV_0300.txt (hunk 2 above)
Comment (Sunaries) [inline QUEST_LV_0300.txt]: Talk to Kupole Meile 

It's Kupole.

File: QUEST_LV_0300.txt [hunk 5]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000055 $Go back to Kupole 메일레
```
Comment (stvkno) [inline QUEST_LV_0300.txt]: QUEST_LV_0300_20150717_00005 -> Go back to Kupol Meile

Comment (Sunaries) [inline QUEST_LV_0300.txt]: Return to Kupole Meile

File: QUEST_LV_0300.txt (hunk 4 above)
Comment (Sunaries) [inline QUEST_LV_0300.txt]: "Check the detectors in this area to help Kupole Meile guide the souls safely from the Stone Frost."

File: QUEST_LV_0300.txt [hunk 6]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000056 $You checked all 4 detectors except the one at the entrance. Now go talk to Kupole 메일레.
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: Now go talk to Kupole 메일레 -> Talk to Kupole Meile

File: QUEST_LV_0300.txt [hunk 7]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000057 $You have something to say to reactivate malfunctioning detector. Talk to Kupole 메일레.
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: Talk to Kupole 메일레 -> Talk to Kupole Meile
to reactivate malfunctioning detector -> to reactivate the malfunctioning detector

File: QUEST_LV_0300.txt [hunk 8]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000021 $Check detector(5)
+QUEST_LV_0300_20150717_000022 $Repairing/MAKING/3/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000023 $Notice/!/Parts are not fixed{nl}You need rope to tighten up!#5
+QUEST_LV_0300_20150717_000024 $Notice/!/You need rope to tighten up#5
+QUEST_LV_0300_20150717_000025 $For Guiding Souls
+QUEST_LV_0300_20150717_000026 $It's okay to be relieved now
+QUEST_LV_0300_20150717_000027 $Checking the device/LOOK/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000028 $Notice/!/As you touch the device, you feel like your soul is escaping!{nl}You are gradually losing energy!
+QUEST_LV_0300_20150717_000029 $Notice/You are recovered now#5
+QUEST_LV_0300_20150717_000030 $All I hope is only...(1)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: All I hope is only...(1) -> Having Hope (1)

File: QUEST_LV_0300.txt [hunk 9]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000031 $About the reason of hoping for death
+QUEST_LV_0300_20150717_000032 $All I hope is only...(3)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: All I hope is only...(3) -> Having Hope (3)

File: QUEST_LV_0300.txt [hunk 10]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000033 $All I hope is only...(2)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: All I hope is only...(2) -> Having Hope (2)

File: QUEST_LV_0300.txt [hunk 11]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
-QUEST_LV_0300_20150717_000053
-QUEST_LV_0300_20150717_000054
-QUEST_LV_0300_20150717_000055
-QUEST_LV_0300_20150717_000056
-QUEST_LV_0300_20150717_000057
-QUEST_LV_0300_20150717_000058
-QUEST_LV_0300_20150717_000059
-QUEST_LV_0300_20150717_000060
+QUEST_LV_0300_20150717_000020 $Repairing/MAKING/3/TRACK_AFTER/None
+QUEST_LV_0300_20150717_000021 $Check detector(5)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: Check detector(5) -> Checking the Detector (5)

File: QUEST_LV_0300.txt [hunk 12]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000033 $All I hope is only...(2)
+QUEST_LV_0300_20150717_000034 $Tell him that you are going to meet Cleric Master
+QUEST_LV_0300_20150717_000035 $You can't cooperate anymore
+QUEST_LV_0300_20150717_000036 $Destroy Terminal(1)
+QUEST_LV_0300_20150717_000037 $About Main Body and Terminal
+QUEST_LV_0300_20150717_000038 $Notice/!/Avoid monsters' attacks and destroy Terminal!#5
+QUEST_LV_0300_20150717_000039 $Destroy Terminal(2)
+QUEST_LV_0300_20150717_000040 $Igniting explosive powder/FIRE/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000041 $Notice/Terminal is destroyed by the explosion!#5
+QUEST_LV_0300_20150717_000042 $Destroy Terminal(3)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: add a space between Terminal and (3)

File: QUEST_LV_0300.txt [hunk 13]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000030 $All I hope is only...(1)
+QUEST_LV_0300_20150717_000031 $About the reason of hoping for death
+QUEST_LV_0300_20150717_000032 $All I hope is only...(3)
+QUEST_LV_0300_20150717_000033 $All I hope is only...(2)
+QUEST_LV_0300_20150717_000034 $Tell him that you are going to meet Cleric Master
+QUEST_LV_0300_20150717_000035 $You can't cooperate anymore
+QUEST_LV_0300_20150717_000036 $Destroy Terminal(1)
+QUEST_LV_0300_20150717_000037 $About Main Body and Terminal
+QUEST_LV_0300_20150717_000038 $Notice/!/Avoid monsters' attacks and destroy Terminal!#5
+QUEST_LV_0300_20150717_000039 $Destroy Terminal(2)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: add a space between Terminal and (2)

File: QUEST_LV_0300.txt [hunk 14]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000027 $Checking the device/LOOK/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000028 $Notice/!/As you touch the device, you feel like your soul is escaping!{nl}You are gradually losing energy!
+QUEST_LV_0300_20150717_000029 $Notice/You are recovered now#5
+QUEST_LV_0300_20150717_000030 $All I hope is only...(1)
+QUEST_LV_0300_20150717_000031 $About the reason of hoping for death
+QUEST_LV_0300_20150717_000032 $All I hope is only...(3)
+QUEST_LV_0300_20150717_000033 $All I hope is only...(2)
+QUEST_LV_0300_20150717_000034 $Tell him that you are going to meet Cleric Master
+QUEST_LV_0300_20150717_000035 $You can't cooperate anymore
+QUEST_LV_0300_20150717_000036 $Destroy Terminal(1)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: add a space between Terminal and (1)

File: QUEST_LV_0300.txt [hunk 15]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000025 $For Guiding Souls
+QUEST_LV_0300_20150717_000026 $It's okay to be relieved now
+QUEST_LV_0300_20150717_000027 $Checking the device/LOOK/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000028 $Notice/!/As you touch the device, you feel like your soul is escaping!{nl}You are gradually losing energy!
+QUEST_LV_0300_20150717_000029 $Notice/You are recovered now#5
+QUEST_LV_0300_20150717_000030 $All I hope is only...(1)
+QUEST_LV_0300_20150717_000031 $About the reason of hoping for death
+QUEST_LV_0300_20150717_000032 $All I hope is only...(3)
+QUEST_LV_0300_20150717_000033 $All I hope is only...(2)
+QUEST_LV_0300_20150717_000034 $Tell him that you are going to meet Cleric Master
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: the Cleric Master

File: QUEST_LV_0300.txt [hunk 16]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000035 $You can't cooperate anymore
+QUEST_LV_0300_20150717_000036 $Destroy Terminal(1)
+QUEST_LV_0300_20150717_000037 $About Main Body and Terminal
+QUEST_LV_0300_20150717_000038 $Notice/!/Avoid monsters' attacks and destroy Terminal!#5
+QUEST_LV_0300_20150717_000039 $Destroy Terminal(2)
+QUEST_LV_0300_20150717_000040 $Igniting explosive powder/FIRE/2/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000041 $Notice/Terminal is destroyed by the explosion!#5
+QUEST_LV_0300_20150717_000042 $Destroy Terminal(3)
+QUEST_LV_0300_20150717_000043 $Eternal Repose(1)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: add a space between Repose and (1)

File: QUEST_LV_0300.txt [hunk 17]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000044 $Last thing to say
+QUEST_LV_0300_20150717_000045 $Eternal Repose(2)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: add a space between Repose and (2)

File: QUEST_LV_0300.txt [hunk 18]
```diff
@@ -17,47 +17,47 @@ QUEST_LV_0300_20150717_000016
...
+QUEST_LV_0300_20150717_000046 $Placing Orb/MAKING/3/TRACK_BEFORE/None
+QUEST_LV_0300_20150717_000047 $Eternal Repose(3)
```
Comment (Sunaries) [inline QUEST_LV_0300.txt]: add a space between Repose and (3)

**Comment by Sunaries** [comment]
Someone else covered your lines in a new PR: https://github.com/Treeofsavior/EnglishTranslation/pull/1341
They are closer to the original meaning.

**Comment by neon279** [comment]
I should follow her translation style.

**Comment by neon279** [comment]
aw... it's actually line 37 to 60 not 59.. how can I edit this?

**Comment by neon279** [comment]
yeah I already changed hope is granted to wish is granted.
I know 것 같습니다 is seems like or looks like. I just omit it by looking previous translations like QUEST_LV_0200.tsv  but yeah I should follow the original text.

**Comment by neon279** [comment]
Last thing to say -> If you have any last words

I didn't quite understand where this text is displayed. If you say so I'll change it

**Comment by HelpfulKoreanGirl** [comment]
Hey @neon279 I incorporated the lines you translated into my pull request (PR#1344) and resolved the conflicts, since we had overlapping lines.

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Comment by HelpfulKoreanGirl** [comment]
@imcgames Why did you merge this one? i edited all of these in my PR#1344. Now I have conflicts again. Did you not like my revisions? Do you want me to just erase the lines that overlapped with neon279's in my PR?

**Comment by Sunaries** [comment]
@HelpfulKoreanGirl My apologies, I gave :+1: to this pull request assuming everything was updated with the merge conflicts and all. :cry:

**Pull request merged**